Makefile

```diff
@@ -14,6 +14,9 @@ rebuild-server:
 server-up:
 	$(COMPOSE_ENGINE) -f docker-compose-dev.yaml up --build --force-recreate -d server
 
+server-down:
+	$(COMPOSE_ENGINE) -f docker-compose-dev.yaml up down
```

Comment:
Probably a typo? The `up down` doesn't look right

Reply:
oh! thanks for noticing, yes I edited that line poorly